frontend/src/pages/collections/[...params].tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
```

Comment:
this is the `/collections/:id` and `/collections/:id/private` page. We need to use `[...params].tsx` as the filename to capture the wildcard matching after `/collections/*`